src/app/modules/auth/__tests__/auth.controller.spec.ts

```diff
@@ -344,4 +362,306 @@ describe('auth.controller', () => {
       expect(mockClearCookie).not.toBeCalled()
     })
   })
+
+  describe('getRedirectLink', () => {
+    const MOCK_REQ = expressHandler.mockRequest({
+      params: {
+        formId: new ObjectId().toHexString(),
+      },
+      query: {
+        isPersistentLogin: true,
+      },
+    })
+    const MOCK_REDIRECT_URL = 'www.mockata.com'
+
+    it('should return 200 with the redirect url when the request is valid and the form has authType SP', async () => {
+      // Arrange
+      const MOCK_FORM = {
```

Comment:
nit: should be in camelCase since it is a declared variable in the test scope and not the describe scope, can also do for the rest of the tests